README.md

```diff
@@ -1,65 +1,38 @@
-There are tons of material available online to know about Git and Github.  
-Some of them are listed below.
+**Feel its too complex to learn?**
 
+Try out this simple youtube playlist.
 
-Git is one of the most widely used tools in programming, so it would definitely be helpful in knowing how to use
-it.  
-
-> At first, if the concepts of Git feel difficult/intimidating, know that it is completely fine to be so.
-Everyone has been there once. Don't let it de-motivate you. 
-You don't have to fully understand/ know everything in a single day.
-
----
-
-Suggested resources for learning Git: 
-
-- [Git and Git for poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) - A YouTube playlist
-    This is one of the easy to follow and quite interesting tutorial video series about Git and GitHub.
+   - [Git and GitHub for poets](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) -
+    This is one of the easiest and quite interesting tutorial video series about Git and GitHub.
     (you may skip the part about GitHub Pages for now if you want)
+    
 
-
-- [Git-It](https://github.com/jlord/git-it-electron) - An interactive Desktop app for learning Git.  <br>
-    For those of you who like to go in a practical route, this would be a great option.  
-    This desktop application takes you over the concepts of Git while allowing you to try out the concepts practically.
-    The [README file](https://github.com/jlord/git-it-electron#what-to-install) has instructions for installing
-    and using it.
-  
-
-After going through the above links, you should now have a GitHub account, and know the basics of working with Git and  have Git set up and installed on your computers.
+> At first, the concepts of Git feel difficult/intimidating, for now that it is completely fine.
+    Everyone has been there once. Don't let it de-motivate you. 
+    You don't have to fully understand/ know everything in a single day.
 
 ---
-## Okay, what next ?
-
-(This section assumes that you have gone through the above links and know the basics of Git and GitHub)
-
-Now that we are familiar with the basics, let's practice our newly learned tool.
-
-Just the steps to be done are outlined here.
-If you have any doubt or stuck on any step, open an issue or reach out.
-
-1. Fork this repo
-2. Clone it locally
-3. Edit it to add your name at the end of the file.
-4. Commit the changes.
-5. Push it to your fork.
-6. Create a PR( Pull Request ) to this repo, so that your changes can be merged here.
-
-
-That's it !! :tada:  
-Now you have successfully learnt the basics of Git and have put your knowledge to use.
-
+  
+# Ready for an Adventure?
+-   Step 1: Install git on your local machines
+-   Step 2: Create a GitHub account
+-   Step 3: Create a Pull request
+-   Step 4: Add < Name - GitHub username > to [Members](https://github.com/opensourcely/git-sandbox/edit/master/README.md) along with the domain you are interested in.
```

Comment:
The link to the file isn't right.

Reply:
Fixed with 9b2a54f19b2fe84b32465bc1976fc8720c8844a9